Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help install test test-unit test-e2e test-e2e-headed lint format clean setup-dev
+.PHONY: help install test test-unit test-e2e test-e2e-headed lint format clean setup-dev build
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)